lib/runner.js

```diff
@@ -49,11 +51,17 @@ app.on('ready', function() {
    */
 
   parent.on('browser-initialize', function(options) {
+    var userPreload = resolve('userPreloadTemplate.js');
```

Comment:
wait, actually what does this do? is the `userPreloadTemplate.js` where you'd put your custom implementation?